### PR TITLE
Falco 0.39.0

### PR DESCRIPTION
**Pull request by alacuku**
This commit adds new config keys introduces in Falco 0.39.0. Furthermore, updates the unit tests for the latest changes in the values.yaml.

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

 /kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

 /area falco-chart

> /area falco-exporter-chart

> /area falcosidekick-chart

> /area event-generator-chart

> /area k8s-metacollector

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Checklist**

- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] CHANGELOG.md updated

### CODE REVIEW

File: charts/falco/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: falco
-version: 4.8.3
-appVersion: "0.38.2"
+version: 4.9.0
+appVersion: "0.39.0-rc1"
```
Comment (Issif) [inline charts/falco/Chart.yaml:4]:
```suggestion
appVersion: "0.39.0"
```

File: charts/falco/template.yaml
Comment (Issif) [inline charts/falco/template.yaml:1]: what's the purpose of this file? btw, the `app.kubernetes.io/version` label has a wrong value in all blocks

Comment (leogr) [reply charts/falco/template.yaml:1]: It looks like a leftover. @alacuku ?

Comment (alacuku) [reply charts/falco/template.yaml:1]: Nice catch @Issif, that's a leftover.

File: charts/falco/Chart.yaml (hunk 1 above)
Comment (alacuku) [reply charts/falco/Chart.yaml]: We are going to use the RC versions until Falco 0.39.0 is out.

File: charts/falco/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: falco
-version: 4.8.3
-appVersion: "0.38.2"
+version: 4.9.0
+appVersion: "0.39.0-rc2"
```
Comment (Issif) [inline charts/falco/Chart.yaml:4]: don't forget to change for 0.39.0 :wink:

**Review by Issif — APPROVED** [review]

**Pull request merged**